piptools/utils.py

```diff
@@ -3,6 +3,7 @@
 import json
 import os
 import shlex
+import typing
```

Comment:
I know I'm late, but importing `typing` directly (and later using `typing.Union` rather than just `Union`) seems unnecessary.